build/linux/installer/conf/td-agent-bit.conf

```diff
@@ -32,9 +32,9 @@
 [INPUT]
     Name tail
     Tag oms.container.log.flbplugin.*
-    Path /var/log/containers/omsagent*.log
+    Path /var/log/containers/ama-logs*.log
     Read_from_Head true
-    DB /var/opt/microsoft/docker-cimprov/state/omsagent-ai.db
+    DB /var/opt/microsoft/docker-cimprov/state/ama-logs-ai.db
```

Comment:
same here. - We will have to keep the same path and file names for the DB to avoid loss the DB state because of the rename.